tasks/Resources/Provisioning/Hive/AWS/oc-provision.robot

```diff
@@ -0,0 +1,56 @@
+*** Settings ***
+Library           OperatingSystem
+
+*** Keywords ***
+Run AWS Configuration In Hive
```

Comment:
If this keyword is used for creating a cluster, could it be renamed to `Create AWS Cluster Using Hive`?